examples/11_rgb_encoding_mono_mobilenet.py

```diff
@@ -56,11 +56,11 @@
 device.startPipeline()
 
 queue_size = 8
-blockingQueue = False # Non blocking host queue - overwrite least recently used frame in queue if it gets full
-q_right = device.getOutputQueue("right", queue_size, blockingQueue)
-q_manip = device.getOutputQueue("manip", queue_size, blockingQueue)
-q_nn = device.getOutputQueue("nn", queue_size, blockingQueue)
-q_rgb_enc = device.getOutputQueue('h265', queue_size, blockingQueue)
+overwriteLRU = True #overwrite least recently used frame in queue if it gets full (not blocking)
```

Comment:
This change would use the previous api (`overwrite` instead of `blocking` naming)

Reply:
fixed